guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -114,6 +114,11 @@ public OrganizationalUnit getOrganizationalUnit( final String name ) {
 
     @Override
     public Collection<OrganizationalUnit> getAllOrganizationalUnits() {
+        return new ArrayList<>( registeredOrganizationalUnits.values() );
+    }
+
+    @Override
+    public Collection<OrganizationalUnit> getOrganizationalUnits() {
```

Comment:
Ah! I got confused by the diff, sorry. (now I also see the javadoc in the interface).